### ISSUE TITLE
Can I debug games from this emulator? 

### ISSUE THREAD

**Issue by L0rdS474n** [libretro/RetroArch#14768]
I love to dissect my games to find flaws and other funny things hidden in the game, I don't want to modify my original boxes to be able to debug so my question is if this emulator can start a dgb server so I can remotely debug this with a debugger of choice?

**Comment by hizzlekizzle** [comment]
Not really. The standalone emulator debuggers are typically the way to go there. [RALibretro](https://docs.retroachievements.org/RALibretro/) is a specialized debugger specifically used for finding RetroAchievements, but it might be useful for your purposes, as well.

**Comment by gouchi** [comment]
@port513 As a reminder from [issue template](https://github.com/libretro/RetroArch/blob/master/.github/ISSUE_TEMPLATE.md)

>Only RetroArch bugs should be filed here. Not core bugs or game bugs
>This is not a forum or a help section, this is strictly developer oriented

So please next time ask your question in the [forum](https://forums.libretro.com/).

For the time being you may close this ticket.

Thank you.

**Comment by zoltanvb** [comment]
This can probably be closed.
